src/Resources/contao/classes/BackendUser.php

```diff
@@ -37,43 +37,43 @@ class BackendUser extends User
 	 * Edit page flag
 	 * @var string
 	 */
-	const CAN_EDIT_PAGE = 1;
+	public const CAN_EDIT_PAGE = 1;
```

Comment:
Not in the old code though. 😄

Reply:
Fixed in d116de00ceb106bb425c771c93ed1925dbb1b59c